src/webargs/core.py

```diff
@@ -65,7 +65,11 @@ class Meta(object):
 
 def is_multiple(field):
     """Return whether or not `field` handles repeated/multi-value arguments."""
-    return isinstance(field, ma.fields.List) and not hasattr(field, "delimiter")
```

Comment:
We probably did this to avoid the circular import. Was a bad decision in retrospect.